### PR TITLE
test: Fix several integration tests for SparkOfflineStore plugin

### PR DESCRIPTION
**Pull request by jbvaningen**
Moved changes from https://github.com/feast-dev/feast/pull/2490

Signed-off-by: Joost van Ingen <joostingen@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by kevjumba** [comment]
@jbvaningen @adchia I think most of the excess changes are probably related to your code linter, it's trying to autoformat. Di d you end up installing our precommit formatter. Merging these changes into master might cause a lot of rebase conflicts with other PRs.

**Comment by achals** [comment]
@jbvaningen I checked out your PR locally and fixed lint

File: sdk/python/tests/integration/registration/test_universal_types.py [hunk 1]
```diff
@@ -158,29 +158,34 @@ def test_entity_inference_types_match(offline_types_test_fixtures):
...
     entity_df["driver_id"] = (
         ["1", "3"] if config.entity_type == ValueType.STRING else [1, 3]
     )
-    now = datetime.utcnow()
-    ts = pd.Timestamp(now).round("ms")
+    ts = pd.Timestamp(datetime.utcnow()).round("ms")
     entity_df["ts"] = [
         ts - timedelta(hours=4),
-        ts - timedelta(hours=2),
+        ts - timedelta(hours=1),
```
[the remark this answers is not in the document]
Comment (kevjumba) [reply sdk/python/tests/integration/registration/test_universal_types.py:185]: I think this is fine, I made the change locally and it looks fine

**Pull request merged**